public/locales/en.json

```diff
@@ -288,6 +288,7 @@
     "postDetails": "Post Details",
     "postTitle1": "Write title of the post",
     "postTitle": "Title",
+    "addMedia": "Upload Photo or Video",
```

Comment:
If you are adding any new text in the project, please make sure to add it for other languages too.

Reply:
Thanks @rishav-jha-mech, i'll add it in different languages🙌